test/integration/style-spec/tests/many-sources.input.json

```diff
@@ -1,14 +0,0 @@
-{
-  "version": 8,
-  "sources": {
-    "one": {
-      "type": "vector",
-      "url": "mapbox://mapbox.mapbox-streets-v6"
```

Comment:
Can't we simply change the address instead of removing the test?

Reply:
As I mentioned in commit message, this test case is only used for check limitation about Mapbox API (Max 15 sources), see https://docs.mapbox.com/api/maps/vector-tiles/#retrieve-vector-tiles